libsrc/leddevice/LedDeviceAdalightApa102.cpp

```diff
@@ -43,7 +35,7 @@ int LedDeviceAdalightApa102::write(const std::vector<ColorRgb> & ledValues)
 		_ledBuffer[5] = _ledBuffer[3] ^ _ledBuffer[4] ^ 0x55; // Checksum
 	}
 
-	for (unsigned iLed=1; iLed<=ledCount; iLed++) {
+	for (signed iLed=1; iLed<=_ledCount; iLed++) {
```

Comment:
If _ledcount is signed (maybe -1 means something in some context) the compiler will throw a warning